pysindy/optimizers/constrained_sr3.py

```diff
@@ -323,25 +269,13 @@ def _update_coef_cvxpy(self, xi, cost, var_len, coef_prev, tol):
             prob = cp.Problem(cp.Minimize(cost))
 
         prob_clone = deepcopy(prob)
-        # default solver is SCS/OSQP here but switches to ECOS for L2
         try:
             prob.solve(
                 max_iter=self.max_iter,
                 eps_abs=tol,
                 eps_rel=tol,
                 verbose=self.verbose_cvxpy,
             )
-        # Annoying error coming from L2 norm switching to use the ECOS
-        # solver, which uses "max_iters" instead of "max_iter", and
-        # similar semantic changes for the other variables.
-        except (TypeError, ValueError):
-            try:
-                prob = prob_clone
-                prob.solve(max_iters=self.max_iter, verbose=self.verbose_cvxpy)
-                xi = prob.variables()[0]
-            except cp.error.SolverError:
-                warnings.warn("Solver failed, setting coefs to zeros")
-                xi.value = np.zeros(var_len)
```

Comment:
Because of the changes in #544, we no longer have l2 norm calculation and we can delete this try-except case